src/utils/Validators.js

```diff
@@ -0,0 +1,44 @@
+class Validators {
+    static CUSTOM_SEPARATOR_REGEX = /^\/\/(.)\n/;
+    static NUMBER_AND_SEPARATOR_REGEX = /^\d+([,:]|\d+)*$/;
+
+    static validateInput(input) {
+        // 1. 빈 입력 검사
+        if (input.trim() === '') {
+            throw new Error('[ERROR] 입력값이 비어 있습니다.');
+        }
+
+        // 2. 커스텀 구분자 확인
+        if (input.startsWith('//')) {
+            const customSeparator = input.match(this.CUSTOM_SEPARATOR_REGEX);
```

Comment:
`input.match`의 결과 중에서 1번 인덱스만 사용한다면 아래처럼 1번 인덱스만 뽑아서 `customSeparator`에 담아도 좋을 것 같습니다!

```suggestion
            const customSeparator = input.match(this.CUSTOM_SEPARATOR_REGEX)[1];
```